src/KnxDevice.cpp

```diff
@@ -499,7 +500,11 @@ void KnxDevice::GetTpUartEvents(e_KnxTpUartEvent event) {
 
 // Static TxTelegramAck() function called by the KnxTpUart layer (callback)
 
-void KnxDevice::TxTelegramAck(e_TpUartTxAck value) {
+void KnxDevice::TxTelegramAck(e_TpUartTxAck 
+#ifdef KNXDevice_DEBUG
+		value
+#endif // KNXDevice_DEBUG
+		) {
```

Comment:
I do not get the point? What's the improval here? In cpp files I always like to have the parameter names.. 

Pls. explain.

Reply:
see comment on src/KonnektingDevice.cpp below
could have added a comment like
#ifdef KNXDevice_DEBUG  // to suppress compiler warning about unused parameter in non debug mode
....